examples/io/viewers/viewer3D-7-planes.cpp

```diff
@@ -30,7 +30,11 @@
 ///////////////////////////////////////////////////////////////////////////////
 #include <cstdlib>
 #include <iostream>
+#ifdef WITH_QT5
+#include <QtWidgets>
+#else
 #include <QtGui/qapplication.h>
+#endif
 #include "DGtal/io/viewers/Viewer3D.h"
```

Comment:
here as commented you can remove line " #include <QtGui/qapplication.h>" and the ifdef